examples/context-mkii.tex

```diff
@@ -25,6 +25,7 @@
     superscripts = yes,
     tableCaptions = yes,
     taskLists = yes,
+    lineBlocks = yes,
```

Comment:
I like to keep the options alphabetically sorted. Adding the options to the end of the list makes it impossible to have several PRs adding new options without collisions on merge.